### PR TITLE
Fixes of i18n on readme.md

### PR DESCRIPTION
**Pull request by GeorgeLimaDev**
#### What problem is this solving?

It relates to task [LOC-10897](https://vtex-dev.atlassian.net/browse/LOC-10897) and KR 23Q2-KR1 ([Review all developer portal documentation in English](https://docs.google.com/document/d/1VduAYpjqdazhyGH5DvD9c52pMu1Xfj4RDx48QNjTFU0/edit)).

[LOC-10897]: https://vtex-dev.atlassian.net/browse/LOC-10897?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -184,47 +183,47 @@ const UISchema = {
...
+| `form`                |
+| `formLoading`         |
+| `formErrorLoading`    |
 | `formSubmitContainer` |
-| `formSubmitButton` |
-| `formErrorServer` |
-| `formErrorUserInput` |
+| `formSubmitButton`    |
+| `formErrorServer`     |
+| `formErrorUserInput`  |
```
Comment (julia-rabello) [inline docs/README.md:227]:
```suggestion
| `formSubmitButton` |
| `formErrorServer` |
| `formErrorUserInput` |
```

**Review by julia-rabello — CHANGES_REQUESTED** [review]

File: docs/README.md [hunk 2]
```diff
@@ -9,36 +9,35 @@ The Store Form app provides blocks responsible for displaying a user form connec
...
 
-```JSON
-dependencies: {
-  "vtex.store-form": "0.x"
-}
-```
+  ```json
+  "dependencies": {
+    "vtex.store-form": "0.x"
+  }
```
Comment (julia-rabello) [inline docs/README.md:21]:
```suggestion
    ```json
    "dependencies": {
        "vtex.store-form": "0.x"
    }
    ```
```

File: docs/README.md [hunk 3]
```diff
@@ -50,7 +49,8 @@ Now, you can use all blocks exported by the `store-form` app. See the full list
      "product-questions-and-answers"
    ]
  },
-```
+...
+}
```
Comment (julia-rabello) [inline docs/README.md:53]: Substituir as linhas 40 até 53 por:

```
    ```json
    {
        "store.product": {
            "children": [
                "flex-layout.row#product-breadcrumb",
                "flex-layout.row#product-main"
                "flex-layout.row#form",
                "shelf.relatedProducts",
                "product-reviews",
                "product-questions-and-answers"
            ]
        },
        ...
    }
    ```
```

**Review by julia-rabello — CHANGES_REQUESTED** [review]

**Review by julia-rabello — APPROVED** [review]

**Comment by GeorgeLimaDev** [comment]
@vtex-apps/store-framework-devs I need a hand with a code owner review. It was requested on the #store-framework Slack channel previously: [Slack message](https://vtex.slack.com/archives/C01PRGM0WG1/p1688564819437709)

**Pull request merged**